packages/module/src/Shortcut/Shortcut.tsx

```diff
@@ -0,0 +1,97 @@
+import * as React from 'react';
+import { MouseIcon } from '@patternfly/react-icons';
+import { Chip } from '@patternfly/react-core';
+import clsx from 'clsx';
+
+export interface ShortcutProps {
+  /** Array of shortcut keys */
+  keys: string[];
+  /** Shortcut description */
+  description?: React.ReactNode;
+  /** Indicates whether symbols should be displayed for certain keys */
+  showSymbols?: boolean;
+  /** Show hover in the shortcut */
+  hover?: boolean;
+  /** Show click in the shortcut */
+  click?: boolean;
+  /** Show right click in the shortcut */
+  rightClick?: boolean;
+  /** Show drag in the shortcut */
+  drag?: boolean;
+  /** Show drag and drop in the shortcut */
+  dragAndDrop?: boolean;
+  /** Shortcut className */
+  className?: string;
+}
+
+const symbols = {
+  'shift': '⇧',
+  'opt': '⌥',
+  'cmd': '⌘',
+  'enter': '↵',
+  'ctrl': '^',
+  'caps lock': '⇪',
+  'tab': '↹',
+  'win': '⊞',
+  'backspace': '⌫'
+}
+
+const Shortcut: React.FunctionComponent<ShortcutProps> = ({
+  keys = [],
+  description = null,
+  showSymbols = true,
+  hover,
+  click,
+  drag, 
+  rightClick, 
+  dragAndDrop,
+  className
+}: ShortcutProps) => {
+  const badges = [
+    ...(hover ? [ 
+      <Chip key="hover" isReadOnly data-test-id="hover">
+        <MouseIcon /> Hover
+      </Chip>
+    ] : []),
+    ...keys.map((key) => {
+      const trimmedKey = key.trim().toLowerCase();
+      return(
+        <Chip key={key} isReadOnly data-test-id={`${key}-button`}>
+          {showSymbols && symbols[trimmedKey] ? `${symbols[trimmedKey]} ` : '' }
+          {key.length === 1 ? key.toUpperCase() : key[0].toUpperCase() + key.slice(1).toLowerCase()}
+        </Chip>
+      )}),
+    ...(click ? [ 
+      <Chip key="click" isReadOnly data-test-id="click">
+        <MouseIcon /> Click
+      </Chip>
+    ] : []),
+    ...(rightClick ? [ 
+      <Chip key="right-click" isReadOnly data-test-id="right-click">
+        <MouseIcon /> Right click
+      </Chip>
+    ] : []),
+    ...(drag ? [ 
+      <Chip key="drag" isReadOnly data-test-id="drag">
+        <MouseIcon /> Drag
+      </Chip>
+    ] : []),
+    ...(dragAndDrop ? [ 
+      <Chip key="drag-and-drop" isReadOnly data-test-id="drag-and-drop">
+        <MouseIcon /> Drag + Drop
+      </Chip>
+    ] : [])
+  ]
+
+  return (
+    <>
+      <span className={clsx({ 'pf-v5-u-mr-lg': description, className })}>
```

Comment:
Do we use the `pf-v5-u` classes in component groups frequently? the styles for the utility classes are not bundled with the standard patternfly.css. It'll require the consumer importing addons.css additionally. If we use those classes frequently, that's fine - but if we can avoid using them by default, that might result in fewer css imports needed by consumers.

Reply:
@nicolethoen for now we have about 20 occurrences in other components. We can replace them with classes using  PF variables. It's probably a better practise

Reply:
I think it'd agree - there may be times where a PF layout component or modifier class might already exist. It's a little bit case by case, but we can seek those out and update them overtime.